packages/form-core/src/tests/FieldApi.spec.ts

```diff
@@ -19,21 +19,31 @@ describe('field api', () => {
     expect(field.getValue()).toBe('test')
   })
 
-  it('should set a value correctly', () => {
```

Comment:
I think we may have accidentally deleted this test. Can we make sure to add it back? :)

Reply:
This is actually a [duplicate test](https://github.com/TanStack/form/blob/main/packages/form-core/src/tests/FieldApi.spec.ts#L22-L54)

Reply:
This is actually a [duplicate test](https://github.com/TanStack/form/blob/main/packages/form-core/src/tests/FieldApi.spec.ts#L22-L54)

Reply:
Oh! You're 100% right. Must've been my bad haha